lib/types.js

```diff
@@ -61,6 +60,24 @@ TInt.prototype = {
 
 exports.TInt = TInt;
 
+function TLong (value) { this.value = value; }
+TLong.prototype = {
```

Comment:
wow... I have not looked at this code in so long I had forgotten it was directly overwriting the prototype... eek.

This should likely be refactored (although this is tangential and has nothing to do with your code)